4] guessing game 3/flask_guessing/flask_server.py

```diff
@@ -0,0 +1,84 @@
+from flask import Flask, render_template
```

Comment:
tady pro to plati v podstate to same co nahore - je to super komplikovane  a neni to spravne